docs/changelog.md

```diff
@@ -2,11 +2,11 @@
 
 ### Added
 
-- [FOGL-8583]: Added user interface support for kvlist with primitive value types(string, integer, float) config item types [#401](https://github.com/fledge-iot/fledge-gui/pull/401)
+- [FOGL-8583]: Added user interface support for kvlist with primitive value types (string, integer, float) config item types [#401](https://github.com/fledge-iot/fledge-gui/pull/401)
```

Comment:
What about list type?

There are list type and kvlist. KVList for object and enum are pending.

Reply:
Do you mean this PR https://github.com/fledge-iot/fledge-gui/pull/397? It is already added there.